pipeline/go.sh

```diff
@@ -572,6 +580,22 @@ function do_replace
 #   do_sync_logs
 }
 
+# do fund distribution 
+function do_fund_dist
+{
+   echo "start to distribute funds to test accounts.. " 
+   local timeout amount
+   timeout=120
+   amount=100000
+   if [ "$CHAIN_ID" == 'testnet' ]; then
+      endpoints=("https://api.s0.os.hmny.io/" "https://api.s1.os.hmny.io/" "https://api.s2.os.hmny.io/")
+   elif [ "$CHAIN_ID" == 'lrtn' ]; then
```

Comment:
the configuration can be found in benchmark-<profile>.json file, check 'rpczone' , and 'network_type'.